clients/da-vinci-client/src/main/java/com/linkedin/davinci/DaVinciBackend.java

```diff
@@ -272,6 +277,15 @@ public DaVinciBackend(
       ingestionService.start();
       ingestionService.addIngestionNotifier(ingestionListener);
 
+      int blobTransferPort = backendConfig.getDvcP2pBlobTransferPort();
```

Comment:
Can we also check the version-level flag to confirm P2P is enabled before creating the blobTransferManager?